test/parallel/test-http2-empty-frame-without-eof.js

```diff
@@ -34,7 +34,9 @@ async function main() {
       client.on('error', common.expectsError(expected));
     }
     stream.resume();
-    await once(stream, 'end');
+    await new Promise((resolve) => {
+      stream.once('close', resolve);
+    });
```

Comment:
Why not `await once(stream, 'close')`?

Reply:
https://github.com/nodejs/node/pull/39607#discussion_r680356478

Reply:
Ah ok `stream.on('error', common.expectsError(expected));` was hidden in the diff.